dnf.spec

```diff
@@ -65,6 +66,7 @@ Provides:       dnf-command(search)
 Provides:       dnf-command(updateinfo)
 Provides:       dnf-command(upgrade)
 Provides:       dnf-command(upgrade-to)
+Conflicts:      dnf-plugins-core < %{min_plugins_core}
```

Comment:
it will not help, you should add conflicts against python2-dnf-plugins-core and python3-dnf-plugins-core